bc_events/session.py

```diff
@@ -55,9 +55,7 @@ def flush(self):
         will not send events and don't have to worry about rolling them back.
         """
 
-        # TODO use a bulk api endpoint once it exists
-        for event in self.events:
-            event.publish()
+        self.publish_all(self.events)
```

Comment:
I had that initially, but thought it was unnecessary because the response times are about the same. 
I've added that logic back in, and also now handle cases where there are more than 250 events in the queue.